apps/02RockPaperScissors.js

```diff
@@ -12,37 +12,37 @@ if (hand1 === hand2) {
 }
 
 if (hand1 === 'rock') {
-   if (hand2 === 'scissors') {
+   else (hand2 === 'scissors') {
```

Comment:
Thank you! I made the corrections and now it works. Awesome!!!

Sent from my iPhone

> On Aug 14, 2016, at 4:12 PM, maddiesolz <notifications@github.com> wrote:
> 
> In apps/02RockPaperScissors.js:
> 
> > @@ -12,37 +12,37 @@ if (hand1 === hand2) {
> >  }
> >  
> >  if (hand1 === 'rock') {
> > -   if (hand2 === 'scissors') {
> > +   else (hand2 === 'scissors') {
> This line 15 is what is showing the error. You cannot have an "else" statement INSIDE of an if statement. this is acceptable:
> 
> javascript```
> if(hand2 === 'scissors') {
> //do something }
> else {
> //do something else
> }
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>